cluster/gce/config-default.sh

```diff
@@ -275,6 +275,14 @@ if [[ ! -z "${NODE_ACCELERATORS}" ]]; then
     fi
 fi
 
+if [[ "${KUBE_FEATURE_GATES:-}" == "AllAlpha=true" ]]; then
```

Comment:
unsure if we need this

Reply:
I added it when I tried to get CRDs installed, but adding this alone does not help.  Let me remove this and see.

Reply:
This is not needed.  Removed now.